core/controllers/suggestion.py

```diff
@@ -344,8 +352,34 @@ def get(self, target_type, suggestion_type):
         """
         self._require_valid_suggestion_and_target_types(
             target_type, suggestion_type)
-        suggestions = suggestion_services.get_reviewable_suggestions(
-            self.user_id, suggestion_type)
+        topic_name = self.request.get('topic_name', None)
+
+        opportunity_summary_exp_ids_specific_to_topic = None
+        if (topic_name is not None) and (topic_name != 'All'):
```

Comment:
'All' should probably be a named constant, right?

Reply:
Yep, I thought so too, and found out we have a constants.ts file which is loaded via constants.py and referred using AppConstants (just also writing here so that you can validate if I found the right file for constants) :) 

But, then I later searched in the codebase and found this: https://github.com/oppia/oppia/blob/develop/core/domain/skill_services.py#L271

which is why I skipped doing it.

Reply:
Yeah, that one's wrong, I'm afraid. Both should use a named constant. (I think this is the issue with bad code making it in -- one instance is not too bad on its own but then the pattern propagates.)

The constants file sounds right. If you push again, please fix it, otherwise we can probably do it in a separate PR. Thanks!